### PR TITLE
IT-3662 | IT-3434 | install vuetify & make DS lib & create input component & create select component 

### PR DESCRIPTION
**Pull request by przemyslawjanpietrzak**

### CODE REVIEW

File: lib/js/atoms/Select/Select.vue [hunk 1]
```diff
@@ -0,0 +1,206 @@
+<template>
+	<div class="a-select" :class="{ '-error': isErrorComputed, '-disabled': disabled }">
+		<ds-app>
+			<v-select
+				:items="['a', 'b', 'c']"
```
Comment (kvas-damian) [inline lib/js/atoms/Select/Select.vue:5]: `items` should be defined as prop

File: lib/js/atoms/Select/Select.vue [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+				:error="isErrorComputed"
+				:readonly="readonly"
+				:disabled="disabled"
+				filled
+			>
+				<template slot="message">
+					<div class="a-select__helpMessage">
+						<icon
+							v-if="isErrorComputed"
+							class="a-input__labelErrorIcon"
```
Comment (kvas-damian) [inline lib/js/atoms/Select/Select.vue:16]: `a-select`

File: lib/js/atoms/Select/Select.vue [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+				</template>
+
+				<template #prepend-inner>
+					<template v-if="leftIcon">
+						<icon :icon="leftIcon" :size="ICON_SIZES.X_SMALL"></icon>
+					</template>
+				</template>
+
+				<template #append>
+					<icon
```
Comment (kvas-damian) [inline lib/js/atoms/Select/Select.vue:33]: do we want rotate this icon when menu is open?

Comment (przemyslawjanpietrzak) [reply lib/js/atoms/Select/Select.vue:33]: I don't know it yet. In current implementation chevron is not visible when select is open

**Review by kvas-damian — APPROVED** [review]

**Comment by kvas-damian** [comment]
Work moved to https://github.com/bethinkpl/design-system/pull/45

**Pull request closed without merging**